### PR TITLE
refactor README.md

### LINKED ISSUE #713: Review and update README
/kind bug
/kind enhancement

[README.md](https://github.com/kubevirt/kubevirt.github.io/blob/source/README.md) is out of date

Please review and update README.md as needed

I suggest we follow the [README.md in user-guide](https://github.com/kubevirt/user-guide/blob/master/README.md) as close as possible

* README should have a clear procedure on how to develop the website
* README should have a clear procedure on how to develop/post content (Blog, Video, Gallery)

### PR DESCRIPTION
**Pull request by eifrach**
Signed-off-by: Eran Ifrach <eifrach@redhat.com+>

**What this PR does**:
*  refactoring the README.md file.
* fix `make run` on Makefile

fixes #713

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -196,7 +198,7 @@ endif
 run: | envvar stop
 	@echo "${GREEN}Makefile: Run site${RESET}"
 	for i in .jekyll-cache _site Gemfile.lock; do rm -rf ./"$${i}" 2> /dev/null; echo -n; done
-	${CONTAINER_ENGINE} run -d --name website --net=host -v ${PWD}:/srv/jekyll:ro${SELINUX_ENABLED} -v /dev/null:/srv/jekyll/Gemfile.lock --mount type=tmpfs,destination=/srv/jekyll/_site --mount type=tmpfs,destination=/srv/jekyll/.jekyll-cache jekyll/jekyll /bin/bash -c "jekyll serve --force_polling --future"
+	${CONTAINER_ENGINE} run -d --name website -p 4000:4000 -v ${PWD}:/srv/jekyll:ro${SELINUX_ENABLED} -v /dev/null:/srv/jekyll/Gemfile.lock --mount type=tmpfs,destination=/srv/jekyll/_site --mount type=tmpfs,destination=/srv/jekyll/.jekyll-cache jekyll/jekyll /bin/bash -c "jekyll serve --trace --force_polling --future"
```
Comment (mazzystr) [inline Makefile:201]: Oh yea!  Thanks for fixing that.  The history was we were going to have user-guide and website cross check each other.  We needed the containers to run `--net=host` to make that happen.

File: README.md [hunk 2]
```diff
@@ -6,43 +6,123 @@
...
+
+
+
+## Join our Community 
+
+* Github Projects: https://github.com/kubevirt
+* Slack: https://kubernetes.slack.com/messages/virtualization
+* Twitter: https://twitter.com/kubevirt
+* Community meetings: [Google Calendar](https://calendar.google.com/calendar/embed?src=18pc0jur01k8f2cccvn5j04j1g%40group.calendar.google.com&ctz=Etc%2FGMT)
+
```
Comment (mazzystr) [inline README.md:125]: I'm not fond of the `Join our community` section being right here.  I don't think the website repo is the entry point into the project.

Can you change to the user-guide `Getting help` and `Developer`?  Let's see how it looks.

**Review by mazzystr — CHANGES_REQUESTED** [review]

**Comment by eifrach** [comment]
@mazzystr  the link test is failing due to in issue in okd.io site. 
the link goes to version 1.5 that is not available anymore

**Pull request merged**